docs/developer/app-store/apps/crm.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: CRM
+sidebar_position: 3
+---
+
+# CRM
+
+## Overview
+
+_CRM_ is a Saleor App that allows providing customer data to external services. Here are the currently available providers:
+
+- [Mailchimp](http://mailchimp.com/)
+- [Twilio Segment](https://segment.com/) [_coming soon_]
+- [RudderStack](https://www.rudderstack.com/) [_coming soon_]
+
+There are two main features CRM App provides
+
+- Initial index of existing customers stored in Saleor and sending them to the provider contact list
+- Creating a new contact in the provider, when a new Customer is created or updated in Saleor
+
+:::caution
+
+To successfully configure the CRM App, you must:
+
+- Have a Mailchimp account
+- Have `MANAGE_APPS` permission
+
+:::
+
+## Installation
+
+The App can be installed from Saleor App Store. To install the App, you must have `MANAGE_USERS` and `MANAGE_APPS` permissions.
+
+## Configuration
+
+The App is configured per provider. You can configure more than one provider - the App will work with at least one provider set up
```

Comment:
```suggestion
The App is configured per provider. You can configure multiple providers - the App will work with at least one provider set up.
```

> The App is configured per provider.  

This sentence is a bit unclear to me. When I first read it, I thought it meant that each app could have only one provider, but it is denied in the next sentence.